resources/assets/lib/notification-maps/message.ts

```diff
@@ -23,7 +23,7 @@ export function messageCompact(item: Notification) {
   let message: string;
 
   const replacements = {
-    content: item.details.content,
+    content: item.details.content || '',
```

Comment:
I wonder if it's better saying no preview or something for those missing data. (also the other one)

Reply:
I'm not sure it'd make sense as the text would show up in between quotes, and I don't think it's worth adding special handling since it'll only show for notifications before the change that haven't been read.

We should also probably look at making things like content, image and url always be required when creating notifications?